### PR TITLE
#156577915 Fix API Tests

### PR DESCRIPTION
**Pull request by rogerokello**
### What does this PR do?
  - Ensure all test files in api/tests have the correct asserts and other tests

#### Description of Task to be completed?
    1. Ensure that all asserts(both present and added) in api/tests are running correctly
    2. Ensure that all tests(both present and added) in api/tests are running correctly

#### How should this be manually tested?
1. Install and configure the application using the readme file

2. Run the following commands

    `cd hc-millennium-falcon`

    `./manage.py test hc/api`

#### What are the relevant pivotal tracker stories? 
#156577915

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,7 @@ hc/local_settings.py
 static-collected
 
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+
+.vscode/
+hc-venv/
```
Comment (emugaya) [inline .gitignore]: Add new line

File: hc/api/tests/test_badge.py [hunk 2]
```diff
@@ -13,12 +13,15 @@ def setUp(self):
 
     def test_it_rejects_bad_signature(self):
         r = self.client.get("/badge/%s/12345678/foo.svg" % self.alice.username)
-        ### Assert the expected response status code
+
```
Comment (emugaya) [inline hc/api/tests/test_badge.py]: Delete line

File: hc/api/tests/test_create_check.py [hunk 3]
```diff
@@ -16,7 +16,13 @@ def post(self, data, expected_error=None):
 
         if expected_error:
             self.assertEqual(r.status_code, 400)
-            ### Assert that the expected error is the response error
+
```
Comment (emugaya) [inline hc/api/tests/test_create_check.py]: Delete line. Do this for the rest of the code.

File: hc/api/tests/test_create_check.py [hunk 4]
```diff
@@ -16,7 +16,13 @@ def post(self, data, expected_error=None):
...
-            ### Assert that the expected error is the response error
+            
+            possible_expected_errors = ["wrong api_key",
+                                        "timeout is not a number",
+                                        "name is not a string",
+                                        "could not parse request body",
+                                        "timeout is too small",
+                                        "grace is too small"]
 
         return r
```
Comment (emugaya) [inline hc/api/tests/test_create_check.py]: What is r?

File: hc/api/tests/test_check_model.py [hunk 5]
```diff
@@ -12,7 +12,12 @@ def test_it_strips_tags(self):
 
         check.tags = " foo  bar "
         self.assertEquals(check.tags_list(), ["foo", "bar"])
-        ### Repeat above test for when check is an empty string
+
+    def test_it_strips_tags_in_empty_string(self):
+        check = Check()
+
+        check.tags = ""
+        self.assertEquals(check.tags_list(), [])
```
Comment (Rwothoromo) [inline hc/api/tests/test_check_model.py]: You could consider:
`
>>> a = ''
>>> not a.strip()
True
>>> a = 'k'
>>> not a.strip()
False
>>> 
`

**Pull request merged**